ui/src/variables/components/VariableForm.test.tsx

```diff
@@ -0,0 +1,84 @@
+// Libraries
+import React from 'react'
+import {shallow} from 'enzyme'
+
+// Components
+import VariableForm from 'src/variables/components/VariableForm'
+
+// Constants
+import {variables} from 'mocks/dummyData'
```

Comment:
Let's decide on a convention on where to put dummy test data.

Reply:
i need a rough overview of this tomorrow. running into weird conditions when trying to run the code in different environments.